api/client/service/opts.go

```diff
@@ -235,7 +235,8 @@ func (m *MountOpt) Type() string {
 func (m *MountOpt) String() string {
 	mounts := []string{}
 	for _, mount := range m.values {
-		mounts = append(mounts, fmt.Sprintf("%v", mount))
+		repr := fmt.Sprintf("%s %s %s", mount.Type, mount.Source, mount.Target)
```

Comment:
Not sure what you mean by fix, as I don't see what's broken about it. But yes, we can easily change this without the user noticing. Currently it is only exists to satisfy an interface and is not actually called outside of test code.